### PR TITLE
Core task 3

### PR DESCRIPTION
**Pull request by LiamCurran95**
Added fetchTopics model and getTopics controller.

Started testing suite - added test for status code 200 and status code 404

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (daviidbartlett) [inline README.md:1]: I'd expect a pieces of info in here by now (check previous tickets)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+describe("/api/topics", () => {
+	describe("GET", () => {
+		test("Status 200 - Body contains an object containing an array of topics", () => {
+			return request(app)
+				.get("/api/topics")
+				.expect(200)
+				.then((res) => {
+					console.log(res.body);
+					res.body.topics.forEach((topic) => {
```
Comment (daviidbartlett) [inline __tests__/app.test.js:20]: remove console.log

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+						expect(topic).toMatchObject({
+							slug: expect.any(String),
+							description: expect.any(String),
+						});
```
Comment (daviidbartlett) [inline __tests__/app.test.js:24]: make sure that you are asserting how many topics are coming back. Could be an empty array 👀

**Review by daviidbartlett — COMMENTED** [review]
few pointers

File: README.md (hunk 1 above)
Comment (LiamCurran95) [reply README.md:1]: Added in - had to do a merge of main / task branch as had got them muddled

File: __tests__/app.test.js (hunk 2 above)
Comment (LiamCurran95) [reply __tests__/app.test.js:20]: Removed

File: __tests__/app.test.js (hunk 3 above)
Comment (LiamCurran95) [reply __tests__/app.test.js:24]: Added in assertion based on test data values

**Pull request merged**